### ISSUE TITLE
core.hooksPath is ignored when using git lfs update --manual

### ISSUE THREAD

**Issue by 4wk-** [git-lfs/git-lfs#5185]
**Describe the bug**
`git lfs update --manual` (and maybe others subcommands as well) does not take `core.hooksPath` variable into account when displaying error/recommended action messages.

**To Reproduce**
1. Having `core.hooksPath` set to `/srv/test/hooks/` with 2 files in it (`post-checkout` and `post-merge`)

2. Try to install git lfs on an existing project : `git lfs install`
> Hook already exists: post-checkout
> [...]
> To resolve this, either:
>  1: run `git lfs update --manual` for instructions on how to merge hooks.
 > 2: run `git lfs update --force` to overwrite your hook.

3. This error is expected. However, now if you run `git lfs update --manual`, the advice is wrong: 

> Add the following to **.git/hooks**/post-checkout:
> #!/bin/sh
> command -v git-lfs >/dev/null 2>&1 || { echo >&2 "\nThis repository is configured for Git LFS but 'git-lfs' was not found on your path. If you no longer wish to use Git LFS, remove this hook by deleting **.git/hooks**/post-checkout.\n"; exit 2; }
> git lfs post-checkout "$@"

**Expected behavior**
The "Add the following to xxx" should NOT be .git/hooks but the value of `core.hooksPath` config variable. Same for the command.
In my example, that mean: 
> Add the following to **[core.hooksPath]**/pre-push:
> [...] remove this hook by deleting **[core.hooksPath]**/post-checkout. [...]

**System environment**
Debian 11.5 dedicated server

**Output of `git lfs env`**
```
git-lfs/2.13.2 (GitHub; linux amd64; go 1.15.9)
git version 2.30.2

Endpoint=https://github.com/REDACTED (auth=none)
  SSH=git@github.com:REDACTED 
LocalWorkingDir=/srv/web/REDACTED 
LocalGitDir=/srv/web/REDACTED/.git
LocalGitStorageDir=/srv/web/REDACTED/.git
LocalMediaDir=/srv/web/REDACTED/.git/lfs/objects
LocalReferenceDirs=
TempDir=/srv/web/REDACTED/.git/lfs/tmp
ConcurrentTransfers=8
TusTransfers=false
BasicTransfersOnly=false
SkipDownloadErrors=false
FetchRecentAlways=false
FetchRecentRefsDays=7
FetchRecentCommitsDays=0
FetchRecentRefsIncludeRemotes=true
PruneOffsetDays=3
PruneVerifyRemoteAlways=false
PruneRemoteName=origin
LfsStorageDir=/srv/web/REDACTED/.git/lfs
AccessDownload=none
AccessUpload=none
DownloadTransfers=basic,lfs-standalone-file
UploadTransfers=basic,lfs-standalone-file
GIT_EXEC_PATH=/usr/lib/git-core
git config filter.lfs.process = "git-lfs filter-process"
git config filter.lfs.smudge = "git-lfs smudge -- %f"
git config filter.lfs.clean = "git-lfs clean -- %f"
```

**Additional context**

**Comment by bk2204** [comment]
Hey,

Thanks for the report.  I agree this should be fixed so that it refers to the correct path.  I'll mark this for inclusion as a bug fix in a future version.

**Comment by bk2204** [comment]
Okay, looking into this some more, we can make the change to the "Add the following" text, but not to the hook text itself.  That's because the way that we determine whether a hook can be updated automatically is to compare it to a list of known values, and if it matches exactly, we can update it; otherwise, we assume the user has customized it and won't modify it.

If we embedded the hook path into the hook itself, it might not match in all cases, and as a result, we'd annoy the user with having to update it manually.  Worse, that tends to break `git lfs install` in automated systems, which we really don't want to do.

What I'm going to do instead here is update the text to mention "the hooks directory (usually .git/hooks)", which will be accurate but not as specific.  That will provide us with the constant text we need without being misleading.

**Comment by 4wk-** [comment]
Hey @bk2204,
Yeah, I think something like "_the hooks directory (defined by `core.hooksPath` config variable, usually .git/hooks)_" would already be a great improvement !
Thank you

**Comment by bk2204** [comment]
This should be fixed by #5245.
